### DISCUSSION TITLE
Asp.Net 6 - CRUD e Entity Framework - Testando a API

### DISCUSSION THREAD

**Discussion by marcelocaselli** [balta-io/forum#356] [Tira Dúvidas] [answered]
Boa noite, após terminar o CRUD, abri o Postman para acompanhar os testes das requisiçoes e ao começar com o 
verbo GET o status code gerou codigo 500....assim como todos os outros verbos continuou o codigo 500.
Revisei toda a Controller o Program.cs e nao achei nenhum erro de sintaxe.

abri o cmd e dei um ping no endpoint, conforme as tentativas abaixo:
PS C:\> ping localhost:5139
A solicitação ping não pôde encontrar o host localhost:5139. Verifique o nome e tente
novamente.
PS C:\> ping http://localhost:5139
A solicitação ping não pôde encontrar o host http://localhost:5139. Verifique o nome e tente
novamente.
PS C:\> ping https://localhost:5139
A solicitação ping não pôde encontrar o host https://localhost:5139. Verifique o nome e tente
novamente.
PS C:\> ping https://localhost:5139/
A solicitação ping não pôde encontrar o host https://localhost:5139/. Verifique o nome e tente
novamente.

Não consegui achar o erro.
![Captura de tela 2023-07-05 234115](https://github.com/balta-io/forum/assets/111652069/474b8496-8aa4-4592-95e9-f66d6447c9f7)

**Comment by marcelocaselli** [comment, marked as answer]
opa acabei de achar o erro......tinha faltado uma } dentro da aspas......e por estar dentro da aspas estava compilando.....valeu obrigado

**Reply by andrebaltieri** [reply]
Showww!
